UPGRADE-6.0.md

```diff
@@ -146,6 +146,11 @@ Inflector
 
  * The component has been removed, use `EnglishInflector` from the String component instead.
 
+Ldap
+----
+
+ * Deprecate `LdapAuthenticator::createAuthenticatedToken()`, use `LdapAuthenticator::createToken()` instead
```

Comment:
We don't do deprecations on `*.0` branches. Shouldn't this PR target 6.1?

Reply:
This is a fix to restore a (deprecated) function removed too fast
To we really need to create the function in 6.0 + deprecate the function in 6.1?

Reply:
> Would making symfony/ldap:6.0 conflict with synfony/security-http < 5.4 fix the issue?

I guess composer would keep `ldap:6.0.5` that allow 5.4 instead of downgrading ldap
+ It will prevent smooth upgrade of the components (ie + upgrading LDAP without upgrading security)

>  I would fix the missing upgrade notes on 5.4, reintroduce the method on 6.0 but internal + throwing, and finally remove the method silently in 6.x

Sounds like a plan